### ISSUE TITLE
ABINIT

### ISSUE THREAD

**Issue by laraPPr** [vscentrum/vsc-software-stack#190]
* link to support ticket: [#2023071960000191](https://otrsdict.ugent.be/otrs/index.pl?Action=AgentTicketZoom;TicketID=125618)
* website: https://www.abinit.org/package/9.10.3
* installation docs: https://docs.abinit.org/installation/
* toolchain: foss/2022b
* easyblock to use: `ConfigureMake`
* required dependencies:
  * [x] netCDF
  * [x] netCDF-Fortran
* optional dependencies:
  * 
* notes:
  * version 9.10.3
  * trivial update of https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/a/ABINIT/ABINIT-9.6.2-intel-2022a.eb
* effort: *(TBD)*
* other install methods
  * conda:  no
  * container image: no
  * pre-built binaries (RHEL8 Linux x86_64): yes (link?) / no
  * easyconfig outside EasyBuild: yes

**Comment by boegel** [comment]
PR with `intel/2022a` available at https://github.com/easybuilders/easybuild-easyconfigs/pull/18462

I initially looked into installing ABINIT 9.10.3 with `intel/2022b`, but it seems like `libxc` 6.x is not supported yet as dependency, even after the existing easyconfig for `libxc` 6.1.0 was updated to not disable building of third and fourth derivates (see ).

Relevant configure output when trying to build ABINIT 9.10.3 on top of libxc 6.1.0 with `intel/2022b`:

```
checking whether to enable LibXC... yes
checking how LibXC parameters have been set... dir
checking for LibXC C preprocessing flags... -I/user/gent/400/vsc40023/eb_arcaninescratch/RHEL8/skylake-ib/software/libxc/6.1.0-intel-compilers-2022.2.1/include
checking for LibXC C flags... none
checking for LibXC linker flags... none
checking for LibXC library flags... -L/user/gent/400/vsc40023/eb_arcaninescratch/RHEL8/skylake-ib/software/libxc/6.1.0-intel-compilers-2022.2.1/lib -lxc
checking whether the LibXC library works... no
checking whether the LibXC library has an old API... no
checking whether the LibXC library has an old API... no
checking whether the LibXC was compiled with Kxc derivatives... no
configure: WARNING: The LibXC package does not provide the
     3rd derivatives of energy! You should recompile it
     using --enable-kxc configure option.
```

https://docs.abinit.org/topics/xc/ mentions "`ABINIT is currently interfaced with LibXC 5`"

**Comment by boegel** [comment]
https://github.com/easybuilders/easybuild-easyconfigs/pull/18462 merged, ABINIT 9.10.3 installed on Hortense, so closing...
